onedal/cluster/kmeans.py

```diff
@@ -337,7 +340,7 @@ def _set_cluster_centers(self, cluster_centers):
     cluster_centers_ = property(_get_cluster_centers, _set_cluster_centers)
 
     def _predict_raw(self, X_table, module, policy, dtype=np.float32):
-        params = self._get_onedal_params(dtype)
+        params = self._get_onedal_params(dtype, result_options="computeAssignments")
```

Comment:
Should default here match daal4py/sklearn default?

Reply:
This is onedal functionality, not necessary to match with sklearn

Reply:
I don't expect any conflict in calling from sklearn or spmd since there will be no user interaction with this parameter. This allows passing separate parameters for train and infer.